deployment_script.py

```diff
@@ -35,6 +36,7 @@ def main():
         print("New db version, execute scripts")
 
         add_scenario_names(mongoDB)
```

Comment:
I'm unclear on when these lines are removed (or commented out) from the deployment script.  Is there a rule of thumb?  And does it matter-- that is, are the scripts assumed to be idempotent (i.e., that re-running them has no effect)?

Part of the reason I ask is because I've already run `add_scenario_names`.